src/systemtest/resources/dateRange/expected/summary.json

```diff
@@ -10,7 +10,10 @@
       },
       "branch": "master",
       "displayName": "reposense/testrepo-Alpha[master]",
-      "outputFolderName": "reposense_testrepo-Alpha_master"
+      "outputFolderName": "reposense_testrepo-Alpha_master",
+      "zoneId": {
+        "id": "Asia/Singapore"
```

Comment:
is there any reasoning in using an object for the `zoneId`? would just a key value pair `"zoneId": "Asia/Singapore" ` suffice?